Library/Homebrew/download_strategy.rb

```diff
@@ -10,6 +10,8 @@
 
 require "mechanize/version"
 require "mechanize/http/content_disposition_parser"
+require "mechanize/http/www_authenticate_parser"
+require "mechanize/http/auth_challenge"
```

Comment:
These files aren't currently included in Git. You'll need to ensure their ignoring is removed from `.gitignore`.

Reply:
Thanks, fixed.